src/App.js

```diff
@@ -181,19 +183,50 @@ class Container extends React.Component {
             </ToggleButtonGroup>
           </div>
         </div>
-        {/* Example usage of MultiButton */}
-        <MultiButton
-          text='Hello'
-          options = {[
-            {text: '1', description: 'Helloo?', audioPath: 'hello_1.wav', default: true},
-            {text: '2', description: 'Hello?', audioPath: 'hello_2.wav'},
-            {text: '3', description: 'HELLO!', audioPath: 'hello_3.wav'},
-            {text: '4', description: 'HELLO!?', audioPath: 'af1_hello_1.wav'},
-            {text: '5', description: 'Hello... is anyone there?', audioPath: 'af1_hello_2.wav'}
-          ]}
-          onMouseUp={this.handleSoundButtonClick}
-          ref={(mb) => this.multiButtons.push(mb)}
-        />
+
+        {/* Main grid */}
+        <Grid
+          container
+          className="MainGrid"
+          direction="row"
+          justify="flex-start"
+          alignItems="flex-start"
+          spacing={2}
+        >
+          {/* Greetings Panel */}
+          <ButtonPanel title='Greetings'>
+            <MultiButton
+              text='Hello'
+              options = {[
+                {text: '1', description: 'Helloo?', audioPath: 'hello_1.wav', default: true},
```

Comment:
No idea what's going on here but I'm only getting sound from `af1_hello_1.wav`. The other selections don't play any sound for me. The anybody there sounds all work though...